dbt/models/iasworld/schema/iasworld.comdat.yml

```diff
@@ -250,9 +253,14 @@ sources:
           - row_values_match_after_join:
               name: iasworld_comdat_class_matches_pardat_class
               column: class
+              column_name: comdat_classes
               external_model: source('iasworld', 'pardat')
               external_column: class
+              external_column_name: pardat_classes
```

Comment:
**nitpick:** The `class_code` sheet already has a `pardat_class` column. Can we rename these to `comdat_class` and `pardat_class`? Same for `asmt_all_classes` in the other test.